drivers/SerialBase.h

```diff
@@ -329,26 +329,25 @@ class SerialBase : private NonCopyable<SerialBase> {
 
 #if DEVICE_SERIAL_ASYNCH
     CThunk<SerialBase> _thunk_irq;
-    DMAUsage _tx_usage;
-    DMAUsage _rx_usage;
+    DMAUsage _tx_usage = DMA_USAGE_NEVER;
+    DMAUsage _rx_usage = DMA_USAGE_NEVER;
     event_callback_t _tx_callback;
     event_callback_t _rx_callback;
-    bool _tx_asynch_set;
-    bool _rx_asynch_set;
+    bool _tx_asynch_set = false;
+    bool _rx_asynch_set = false;
 #endif
-
-    serial_t         _serial;
+    serial_t         _serial {};
```

Comment:
Could we avoid doing the structure reorder now? Could conceivably cause some binary compatibility breakage - unlikely, but if it's not necessary, let's avoid it.

Reply:
Ah, you put it back. But maybe squash this together - the two commits move them and put them back.

Reply:
You mean the re-location of the #if DEVICE_SERIAL_FC block?

Reply:
Yes, the two commits move it and put it back, which is just history noise.

Reply:
Yes, indeed. It's now squashed and force pushed.